third-party/rsutils/include/rsutils/byte-manipulation.h

```diff
@@ -0,0 +1,13 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+namespace rsutils
+{
+    namespace byte
+    {
+        unsigned char reverse_bits(unsigned char b);

```

Comment:
uint8_t instead of char
Both in function header and inside

Reply:
Lets also place it in `number` folder and in number namespace
No need for new ones